### ISSUE TITLE
Search in Sphinx when hosted on twistedmatrix doesn't work

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8341]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8341|
|Type|defect|
|Created|2016-05-20 04:07:43Z|
|Branch|https://github.com/twisted/twisted/tree/noquery-8341|

This is because it uses the Trac version of jQuery, I believe -- we ought to not use that, and use Sphinx's version.

<details><summary>Searchable metadata</summary>

```
trac-id__8341 8341
type__defect defect
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__noquery_8341 noquery-8341
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__approved approved
time__1463717263952601 1463717263952601
changetime__1463912006800096 1463912006800096
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Looking good, please ignore the topfile builder ([#8336](https://github.com/twisted/twisted/issues/8336)#ticket) hasnt landed yet.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @hawkowl|
|-|-|

Looks good to me, delete more code ;-).

Please land.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_5"></a>|@tomprince commented|
|-|-|

[#8341](https://github.com/twisted/twisted/issues/8341) is a related ticket. We should perhaps update existing documentation with this fix as well, once this ticket lands.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 99a2af099c6dc06ac05b10a9d18838138929df20
```
#!CommitTicketReference repository="" revision="99a2af099c6dc06ac05b10a9d18838138929df20"
Merge noquery-8341: Fix Sphinx hosted on twistedmatrix.com

Author: hawkowl
Reviewer: glyph
Fixes: #8341
```
